frodo.js

```diff
@@ -1,58 +1,58 @@
 /**
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!